bin/run_atarchiver.sh

```diff
@@ -4,28 +4,42 @@ source $loc/base_init.sh
 
 if [ -z $1 ];
 then
-    echo "$0: missing arguments: lsst_dds_partition_prefix container_version"
+    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
     exit 1
 fi
 
 if [ -z $2 ];
 then
-    echo "$0: missing arguments: lsst_dds_partition_prefix container_version"
+    echo "$0: missing argument: lsst_dds_partition_prefix container_version"
     exit 1
 fi
 
 lsst_dds_partition_prefix=$1
 container_version=$2
+network_interface_address=$3
+domain_id="0"
+if [ -n $4 ];
+then
+    domain_id=$4
+fi
```

Comment:
Assigning a default value to a variable if the explicit value is not provided could also be done with
```bash
domain_id=${4:-0}
```
However, at the end of the day it's just a matter of personal preferences, so I'm leaving the decision which one to use up to you.

Reply:
I usually do it that way to make it a bit more clear what's being done.